corehq/apps/app_manager/static/app_manager/js/releases.js

```diff
@@ -398,7 +398,7 @@ hqDefine('app_manager/js/releases.js', function () {
                     } else if (self.lastAppVersion() !== self.currentAppVersion()) {
                         self.actuallyMakeBuild();
                     } else {
-                        window.alert(gettext("No new changes to deploy!"));
+                        window.alert(gettext("No new changes!"));
```

Comment:
really? what would you want to happen? suppose we could not show the "Make New Version" button if you don't have changes...but this is one of the few pages that works reasonably well if you're making changes in other tabs.